src/Microsoft.DncEng.SecretManager/appsettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ApplicationInsightsInstrumentationKey": "00000000-0000-0000-0000-000000000000"
```

Comment:
? zero?

Reply:
We haven't created an app insights for this yet. I will add the key here once we have one, but I wanted to get the projects in so we can start work on implementations.